changelog/unreleased/kong/bump-atc-router.yml

```diff
@@ -0,0 +1,5 @@
+message: |
```

Comment:
```suggestion
message: >
```

More details: https://yaml-multiline.info/

Reply:
I found most of our change logs use "|" instead of ">".